docs/cluster-deployment/deploy-to-cluster-aws/signup-aws.rst

```diff
@@ -23,7 +23,7 @@ Using the CrateDB Cloud offer on AWS Marketplace
 
 To subscribe to CrateDB Cloud's offering on AWS Marketplace, find the CrateDB
 Cloud offer page. You can either search "CrateDB Cloud" in the search bar at
-the top of the AWS Marketplace front page, or follow `this link`_ to the offer.
+the top of the AWS Marketplace front page, or follow `this link to the offer`_.
```

Comment:
I noticed you improving some of the links in this doc! looks good!

one minor nit: I said it's a better style if links use noun phrases, and using "this link" (or similar) DOES technically count (haha) but then the noun is pointing to the link itself and not the destination the link takes you to. perhaps think of the link text like the label you would put on a map? it should describe the location, not the method of travel ^_^

so in this case, how about:

> You can either search "CrateDB Cloud" in the search bar at the top of the AWS Marketplace front page or go directly to `the offer`_.

Reply:
updated https://github.com/crate/crate-docs/blob/master/style/hypertext.rst